pkg/images/enricher/enricher_impl.go

```diff
@@ -224,18 +224,25 @@ func (e *enricherImpl) updateImageWithExistingImage(image *storage.Image, existi
 
 // EnrichImage enriches an image with the integration set present.
 func (e *enricherImpl) EnrichImage(ctx context.Context, enrichContext EnrichmentContext, image *storage.Image) (EnrichmentResult, error) {
-	if shouldDelegate, err := e.delegateEnrichImage(ctx, enrichContext, image); shouldDelegate {
-		// This enrichment should have been delegated, short circuit.
-		if err != nil {
+	shouldDelegate, err := e.delegateEnrichImage(ctx, enrichContext, image)
+	var delegateErr error
+	if shouldDelegate {
```

Comment:
nit: can revert this back to its original state now that delegatedErr is the only thing needed outside the block

```suggestion
	var delegateErr error
	if shouldDelegate, err := e.delegateEnrichImage(ctx, enrichContext, image); shouldDelegate {
```

Reply:
I will address this in another PR so I don't have to rebuild and re-manual test😄